ansible/roles/common/tasks/common.yml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: disable swap since kubernetes can't work with swap enabled (1/2)
-  command: swapoff -a
-  when: ansible_swaptotal_mb > 0
+# - name: disable swap since kubernetes can't work with swap enabled (1/2)
+#   command: swapoff -a
+#   when: ansible_swaptotal_mb > 0
```

Comment:
```suggestion
- name: disable swap since kubernetes can't work with swap enabled (1/2)
   command: swapoff -a
   when: ansible_swaptotal_mb > 0
```